### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by secondsun**
A bunch of Minor updates for Java 8, aerogear-parent, and android exclusions.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
     <groupId>org.jboss.aerogear</groupId>
     <artifactId>aerogear-crypto</artifactId>
-    <version>0.1.5-SNAPSHOT</version>
+    <version>0.1.6-SNAPSHOT</version>
```
Comment (matzew) [inline pom.xml]: why the version bump here ?

File: pom.xml [hunk 2]
```diff
@@ -303,6 +303,7 @@
                                                 <exclude>META-INF/*.SF</exclude>
                                                 <exclude>META-INF/*.DSA</exclude>
                                                 <exclude>META-INF/*.RSA</exclude>
+                                                <exclude>META-INF/maven/com.madgag/**</exclude>
```
Comment (matzew) [inline pom.xml]: ah! :)

File: pom.xml (hunk 1 above)
Comment (secondsun) [inline pom.xml]: Because 0.1.5 is the currently released version

Comment (matzew) [inline pom.xml]: I see, some one did an improper release before :-)

Comment (secondsun) [inline pom.xml]: You do the git blame, I'll get the pitchforks.

Comment (matzew) [inline pom.xml]: LOL

**Comment by matzew** [comment]
@secondsun is this PR complete? On IRC you mentioned a it needs an update, so I am not sure

**Pull request merged**